hypothesis/lms
# Use manifest-building function from @hypothesis/frontend-build

Replace the asset manifest-building code in scripts/gulp/manifest.js
with the `generateManifest` function from the @hypothesis/frontend-build
package.

---

**Testing:**

1. Remove the `build` directory
2. Run `gulp build` and check that `build/manifest.json` is generated and has the correct format and files (there should be an entry for all JS, CSS and sourcemap files in the `build/` directory)
